packages/headless/src/api/apolloClient.ts

```diff
@@ -102,13 +91,20 @@ function createApolloClient(
  * }
  * ```
  */
-export function initializeApollo(
-  context?: NextPageContext | GetStaticPropsContext | GetServerSidePropsContext,
-  initialState = null,
+export function getApolloClient(
+  context?: PersistentContext,
+  initialState: unknown = null,
 ): ApolloClient<NormalizedCacheObject> {
-  const localApolloClient = createApolloClient({
-    cookies: getCookiesFromContext(context),
-  });
+  let localApolloClient:
+    | ApolloClient<NormalizedCacheObject>
```

Comment:
@wjohnsto hi there, just a random passerby here, but is this a typical union with a prepended `|` or is there something more going on that I may be misunderstanding?

Reply:
I don't think there is anything more going on, it's possible this was the output from eslint/prettier as I do not remember writing this specifically. The outcome is the same as `let localApolloClient: ApolloClient<NormalizedCacheObject>`.

Reply:
Awesome, thanks for clearing that up!